src/GraphWebhooks/Models/SubscriptionRecord.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+namespace GraphWebhooks.Models
+{
+    /// <summary>
+    /// Subscription record saved in subscription store
+    /// </summary>
+    public class SubscriptionRecord
```

Comment:
you could use a record here https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records

Reply:
Hmm that's cool, I hadn't heard of this before. It doesn't look like I'm actually using this in anyway that would benefit from declaring it as a record, but it is definitely used just to store data.